docs/en_US/AnnotationSpec.md

```diff
@@ -36,8 +36,8 @@ There are 10 types to express your search space as follows:
 
 * `@nni.variable(nni.choice(option1,option2,...,optionN),name=variable)`
   Which means the variable value is one of the options, which should be a list The elements of options can themselves be stochastic expressions
-* `@nni.variable(nni.randint(upper),name=variable)`
-  Which means the variable value is a random integer in the range [0, upper).
+* `@nni.variable(nni.randint(lower, upper),name=variable)`
```

Comment:
lower and upper should be integers? or could be float?

Reply:
Could you also check whether the chosen value is integer or float?

Reply:
I asked this because users may want to know the value's type, for example, an integer cannot be used as array index. It would be better to explicitly tell users.

Reply:
The check may in another PR.